package.json

```diff
@@ -8,18 +8,18 @@
     "build:renderer": "cross-env NODE_ENV=production webpack --config ./.erb/configs/webpack.config.renderer.prod.babel.js",
     "rebuild": "electron-rebuild --parallel --types prod,dev,optional --module-dir src",
```

Comment:
I think `optional --module-dir src` should change to `optional --module-dir build/app`